lib/widgets/action_sheet.dart

```diff
@@ -60,6 +71,49 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
   }
 }
 
+// This button is very temporary, to complete #125 before we have a way to
+// choose an arbitrary reaction (#388). So, skipping tests and i18n.
+class AddThumbsUpButton extends MessageActionSheetMenuItemButton {
+  AddThumbsUpButton({
+    super.key,
+    required super.message,
+    required super.messageListContext,
+  });
+
+  @override get icon => Icons.add_reaction_outlined;
+
+  @override
+  String label(ZulipLocalizations zulipLocalizations) {
+    return 'React with 👍'; // skip translation for now
```

Comment:
```suggestion
    return 'React with 👍'; // TODO(i18n) skip translation for now
```

I'd like this spot to come up in a grep for places that need i18n, even if we don't end up acting on this one in particular before we replace the code.